testing/fake-diff-editor.rs

```diff
@@ -30,7 +30,7 @@ struct Args {
     after: PathBuf,
 }
 
-fn files_recursively(dir: &Path) -> HashSet<String> {
+fn files_recursively(dir: &PathBuf) -> HashSet<String> {
```

Comment:
nit: I thought Clippy would complain about `&PathBuf`, suggesting to use `&Path` instead. Is that not the case? I see that the Clippy check passes on CI, but I'm not sure I've set that up correctly after a somewhat recent change to it. Also, changing to `&PathBuf` seems unrelated to the other changes in the commit.

Reply:
No idea why this happened. I un -did it.